cmd/create/createCluster.go

```diff
@@ -0,0 +1,419 @@
+/*Package create ...
+
+Copyright © 2020 The k3d Author(s)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+package create
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	cliutil "github.com/rancher/k3d/cmd/util"
+	"github.com/rancher/k3d/pkg/cluster"
+	k3dCluster "github.com/rancher/k3d/pkg/cluster"
+	"github.com/rancher/k3d/pkg/runtimes"
+	k3d "github.com/rancher/k3d/pkg/types"
+	"github.com/rancher/k3d/version"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// NewCmdCreateCluster returns a new cobra command
+func NewCmdCreateCluster() *cobra.Command {
+
+	createClusterOpts := &k3d.CreateClusterOpts{}
+
+	// create new command
+	cmd := &cobra.Command{
+		Use:   "cluster NAME",
```

Comment:
Can we add a `--wait` flag?  The biggest headache of automating a k3d based setup is know when the cluster is up, ready, and get-kubeconfig won't fail.

Reply:
The `--wait` flag exists already (see https://github.com/rancher/k3d/pull/212/files/33d54132edf05a86849ade7c8f60defbdfcf6f14#diff-17f115421f9598dc6eca63780144c2d2R80 - Line 80) :+1: 
Or would you make it the default?

Reply:
Clarified in a chat